docs/contents/dev-docs/testing.md

```diff
@@ -246,3 +248,57 @@ make delete-all-kind-clusters
 
 With this the testing is completed. Thanks for your time and we appreciate your
 feedback.
+
+## Installing helm charts from ECR repository and test on Amazon EKS cluster
+This section contains steps to test ACK service controller using helm chart and Amazon EKS cluster.
+These steps take ACK ElastiCache controller as example and require Amazon EKS cluster. 
+
+Refer Amazon EKS [documentation](https://docs.aws.amazon.com/eks/index.html) to get details on creating Amazon EKS cluster.
+
+Depending upon [Amazon EKS cluster endpoint access control](https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html)
+setup for the Amazon EKS cluster, following steps are to be executed from a host that can access the EKS cluster endpoint used for testing.
+
+### Clone ACK `community` repository
+ACK [community](https://github.com/aws-controllers-k8s/community) repository contains scripts used for testing.
+
+Clone ACK `community` repository on the test runner host.
+```bash
+# cd <directory to run the tests from>
+git clone https://github.com/aws-controllers-k8s/community.git community
+cd community
+```
+
+### Setup IRSA
+
+If no IAM OIDC provider is set for Amazon EKS cluster, follow the instructions to set it: 
+[Create an IAM OIDC provider for your cluster ](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html)
+For example:
+```bash
+EKS_CLUSTER_NAME=<eks cluster name>
+eksctl utils associate-iam-oidc-provider --cluster $EKS_CLUSTER_NAME --approve
+```
+The above example uses `eksctl` utility. To install it, follow: [installing-eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html#installing-eksctl)
+
+Once the IAM OIDC provider is set for Amazon EKS Cluster, setup IRSA on Amazon EKS cluster:
+```bash
+# cd community
+export AWS_REGION=us-west-2
+export EKS_CLUSTER_NAME="eks_cluster_name"
+export IAM_POLICY_ARN='arn:aws:iam::aws:policy/AmazonElastiCacheFullAccess'
+export SERVICE=elasticache 
+make eks-setup-irsa
+```
+
+For details on IRSA setup, refer [IRSA setup on Amazon EKS cluster](../../irsa.md) document and 
```

Comment:
Placing this at the top of the section to provide more context (for concepts like the OIDC provider) could be useful.